### DISCUSSION TITLE
TAP (Bridge) support

### DISCUSSION THREAD

**Discussion by edricus** [angristan/openvpn-install#1105] [Ideas]
TAP interface is really useful to include multiple machines on the server side without manually adding routes.  
This is the purpose of using a VPN for a lot of people.  
I don't know how it's not a feature yet, the script is really great and do everything but this. Why ?  
You can read all the usage of TAP here : [https://serverfault.com/questions/21157/should-i-use-tap-or-tun-for-openvpn](https://serverfault.com/questions/21157/should-i-use-tap-or-tun-for-openvpn)  
I hope to hear about this someday, thanks considering.

**Comment by Wheedman** [comment]
I would also love to see a tap support so the vpn can be used as layer 2 vpn for game discovery and an online lan party with the good old games.
